Taskfile.yml

```diff
@@ -287,27 +292,27 @@ tasks:
   package-venv:
     internal: true
     vars:
-      CHECKSUM_FILE: "{{.BUILD_DIR}}/{{.TASK}}.md5"
-      OUTPUT_DIR: "{{.PACKAGE_VENV_DIR}}"
+      CHECKSUM_FILE: "{{.G_BUILD_DIR}}/{{.TASK}}.md5"
+      OUTPUT_DIR: "{{.G_PACKAGE_VENV_DIR}}"
     cmds:
-      - "rm -rf '{{.OUTPUT_DIR}}'"
-      - "python3 -m venv '{{.OUTPUT_DIR}}'"
-      - |-
-        . "{{.OUTPUT_DIR}}/bin/activate"
-        pip3 install --upgrade -r "{{.TASKFILE_DIR}}/requirements.txt"
+      - task: "create-venv"
+        vars:
+          LABEL: "package"
+          OUTPUT_DIR: "{{.OUTPUT_DIR}}"
+          REQUIREMENTS_FILE: "{{.ROOT_DIR}}/requirements.txt"
```

Comment:
For consistency, I think we can get rid of the {{.ROOT_DIR}} here, given `dir` is not configured. Or we could append the same to the one in task "venv" in lint-tasks.yml.

Reply:
Good point